pkg/server/endpoints/authorized_entryfetcher_attested_nodes.go

```diff
@@ -111,8 +113,8 @@ func (a *attestedNodes) updateCache(ctx context.Context) error {
 		// were skipped over and need to be queued in case they show up later.
 		// This can happen when a long running transaction allocates an event ID but a shorter transaction
 		// comes in after, allocates and commits the ID first. If a read comes in at this moment, the event id for
-		// the longer running transaction will be skipped over
-		if !a.firstEventTime.IsZero() && event.EventID != a.lastEventID+1 {
+		// the longer running transaction will be skipped over.
+		if !a.firstEventTime.IsZero() {
```

Comment:
Following same style suggested by this [code review comment](https://github.com/spiffe/spire/pull/5289#discussion_r1683227127).